src/HistoryQuery/HistoryQuery.class.js

```diff
@@ -0,0 +1,181 @@
+const ProtocolFactory = require('../south/ProtocolFactory.class')
+
+class HistoryQuery {
+  static STATUS_PENDING = 'pending'
```

Comment:
maybe a comment to explain the different status

Reply:
fixed